training/xgboost/structured/base/README.md

```diff
@@ -0,0 +1,151 @@
+# xgboost Estimator - Trainer Package
+
+## Overview
+
+The purpose of this directory is to provide a sample for how you can package an xgboost
+training model to submit it to AI Platform. The sample makes it
+easier to organize your code, and to adapt it to your dataset. In more details,
+the template covers the following functionality:
+
+*   Metadata to define your dataset, features, and target.
+*   Standard implementation of input, parsing, and serving functions.
+*   Train, evaluate, and export the model.
+
+Although this sample provides standard implementation to different
+functionality, you can customize these parts with your own implementation.
+
+## Prerequisites
+
+* Follow the instructions in the [setup](../../../../setup) directory in order to setup your environment
+* Follow the instructions in the [datasets](../../../../datasets) directory and 
+run [download-taxi.sh](../../../../datasets/download-taxi.sh) to download the datasets
+* Create a Python 3 virtual environment and activate it
```

Comment:
Doesn't the setup directory make you do this.

Reply:
Good point. Removing it.